src/BuildScriptGenerator/BaseBashBuildScriptProperties.cs

```diff
@@ -35,8 +35,6 @@ public class BaseBashBuildScriptProperties
         /// </summary>
         public string PostBuildScriptPath { get; set; }
 
-        public IEnumerable<string> DirectoriesToExcludeFromCopyToBuildOutputDir { get; set; }
```

Comment:
Removed this from common base bash script because each platform will handle it explicitly